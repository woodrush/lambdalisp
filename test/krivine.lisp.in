"01010000011101000100010"