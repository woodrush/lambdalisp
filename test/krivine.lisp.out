> 
Input: 01010000011101000100010
Lexed: (A A L L A 1 0 L 0 L 0)
Parsed: (((L L 1 0) (L . 0)) (L . 0))
Krivine machine transitions:
----
t: (((L L 1 0) (L . 0)) (L . 0))
p: ()
e: ()
----
t: ((L L 1 0) (L . 0))
p: (((L . 0)))
e: ()
----
t: (L L 1 0)
p: (((L . 0)) ((L . 0)))
e: ()
----
t: (L 1 0)
p: (((L . 0)))
e: (((L . 0)))
----
t: (1 0)
p: ()
e: (((L . 0)) ((L . 0)))
----
t: 1
p: ((0 ((L . 0)) ((L . 0))))
e: (((L . 0)) ((L . 0)))
----
t: (L . 0)
p: ((0 ((L . 0)) ((L . 0))))
e: ()
----
t: 0
p: ()
e: ((0 ((L . 0)) ((L . 0))))
----
t: 0
p: ()
e: (((L . 0)) ((L . 0)))
----
t: (L . 0)
p: ()
e: ()
----
Result: (L . 0)
